streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -547,7 +549,8 @@ public void shouldCountAndMaterializeResults() {
 
         processData();
 
-        final KeyValueStore<String, Long> count = (KeyValueStore<String, Long>) driver.allStateStores().get("count");
+        final KeyValueStore<String, Long> count = driver.getKeyValueStore("count");
+        Map<String, StateStore> allStateStores = driver.getAllStateStores();
```

Comment:
Why do we need this? It seems `allStateStores` is never used?

Reply:
Yeah, we don't need it. I'll remove it.